src/main/java/io/usethesource/vallang/impl/reference/Map.java

```diff
@@ -92,4 +93,14 @@ public int hashCode() {
 		return MapFunctions.hashCode(getValueFactory(), this);
 	}
 	
+	@Override
```

Comment:
That method can be removed, because a default implementation is available in `AbstractMap.java`.